backend/src/main/java/ee/tlu/forum/ForumApplication.java

```diff
@@ -10,4 +21,53 @@ public static void main(String[] args) {
 		SpringApplication.run(ForumApplication.class, args);
 	}
 
+	@Bean
```

Comment:
I *think* I understand what's going on here but mainly, I'm just trusting that you know what you're doing 😉